### DISCUSSION TITLE
Anyone can tell why I have to move large file from one to another again and again !! Really stupid design!

### DISCUSSION THREAD

**Discussion by seantw1974** [filecoin-project/lotus#7060] [Storage Provider] [answered]
Anyone can tell why I have to move large file from one to another again and again !! Really stupid design!

**Comment by johnwick147** [comment]
use storx

**Comment by johnwick147** [comment]
https://storx.tech

**Comment by LexLuthr** [comment, marked as answer]
@seantw1974 Are you referring to moving files between worker during the sealing phase? If yes, then it has been fixed with the latest release. 
https://github.com/filecoin-project/lotus/pull/7453
https://lotu.sh/docs/storage-providers/seal-workers/
